File: Hospital.Web/src/models/Appointment.ts
import { Doctor } from "./Doctor";
import { Office } from "./Office";

export interface Appointment {
    id: number,
    doctorId: number,
    doctor: Doctor,
    officeId: number,
    office: Office,
    startDate: Date,
    endDate: Date,
    patientName: string
}

File: Hospital.Host/Hospital.BusinessLogic/Models/Entities/Appointment.cs
﻿using System;

namespace Hospital.BusinessLogic.Models.Entities
{
    public class Appointment
    {
        public int Id { get; set; }

        public int DoctorId { get; set; }

        public Doctor Doctor { get; set; } = null!;

        public int IntervalId { get; set; }

        public Interval Interval { get; set; } = null!;

        public int OfficeId { get; set; }

        public Office Office { get; set; } = null!;

        public DateTime Date { get; set; }

        public string PatientName { get; set; } = null!;
    }
}
